ramwong/S381F-project
# Test no url

[22:36]
login test
1. wrong username with password
2. wrong username without password 
3. true username with password
4. true username without password
expect: only test4 can login
passed

[22:39]
/read test
1. load without collection - success
2. load with collection -success
3. load without docs -success
4. load with 1 doc - success
5. load with some docs - success
6. redirect to home without session - success

[22:39]
/new  test
1. load page - success
2. redirect to home without session - success

[22:43]
/processcreate test
1. create without collection - success
2. check cannot create with empty name - success
3. create with all empty fields except name -success
4. create with all fields - success
5. redirect to home without session and not create - success

[22:44]
/display test
1. load with picture- success
2. load without picture - success
3. redirect to home without session - success
4. no gps=no map link - success

[22:47]
/rate test
1. load page - success
2. check can only submit between 1-10 - success
3. redirect to home without session - success

[22:48]
/processrate test
1. check rate once only - success
2. redirect to home without session and not update - success
3. check others can rate -success
4. check others can rate after another rated -success

[22:49]
/change test
1. load page - success
2. fields auto-filled in - success
3. redirect to home without session - success
4. check can only load by owner - success

[22:53]
/processchange test
1. check cannot empty name - success
2. change exists field - success
3. update empty field - success
4. update image (no image before) -success
5. update image(have image before) - success
6. redirect to home without session and not update - success

[22:55]
/processdelte test
1. delete -success
2. page load - success
3. check can only remove by owner - success
4. redirect to home without session and not remove - success

api name test
1. curl -X GET http://localhost:8099/api/restaurant/name/name with docs  -success
2. curl -X GET http://localhost:8099/api/restaurant/name/name without docs -success
3. curl -X GET http://localhost:8099/api/restaurant/name/name with not exist docs -success

api borough test
1. curl -X GET http://localhost:8099/api/restaurant/borough/borough with docs  -success
2. curl -X GET http://localhost:8099/api/restaurant/borough/borough without docs -success
3. curl -X GET http://localhost:8099/api/restaurant/borough/borough with not exist docs -success

api cuisine test
1. curl -X GET http://localhost:8099/api/restaurant/cuisine/cuisine with docs  -success
2. curl -X GET http://localhost:8099/api/restaurant/cuisine/cuisine without docs -success
3. curl -X GET http://localhost:8099/api/restaurant/cuisine/cuisine with not exist docs -success
